zygrader/ui/layers.py

```diff
@@ -579,7 +580,15 @@ def set_searchable(self, prompt: str, search_fn=__string_search_fn):
     def set_sortable(self):
         self.component.set_sortable()
 
+    def set_paged(self):
+        self.component.set_paged()
+        self._paged = True
```

Comment:
As mentioned in components, reversibility could make sense.
```suggestion
    def set_paged(self, paged=True):
        self.component.set_paged(paged)
        self._paged = paged
```

Reply:
And as mentioned elsewhere, there's no use case for it, so we don't need to (and perhaps even *shouldn't*) add this suggestion.